src/redux/data.ts

```diff
@@ -875,17 +876,18 @@ const checkForUpdatedNonce = (transactionData: ZerionTransaction[]) => (
  *
  * @param removedTransactions Removed transaction data.
  */
-const checkForRemovedNonce = (removedTransactions: ZerionTransaction[]) => (
-  dispatch: ThunkDispatch<AppState, unknown, never>
-) => {
-  const txSortedByAscendingNonce = removedTransactions.sort(
-    ({ nonce: n1 }, { nonce: n2 }) => (n1 ?? 0) - (n2 ?? 0)
-  );
-  const [lowestNonceTx] = txSortedByAscendingNonce;
-  // @ts-expect-error `ZerionTransaction` doesn't have a `network` field, but
-  // the undefined network is defaulted to mainnet later.
-  const { address_from, network, nonce } = lowestNonceTx;
-  dispatch(decrementNonce(address_from!, nonce!, network));
+const checkForRemovedNonce = (
```

Comment:
can removed txns be ones that the `accountAddress` did not initiate?

Reply:
Have we ever guarded against this case in the past? I added the filter. Lmk if this is what you had in mind.